hooks/useStakingData.ts

```diff
@@ -212,28 +218,27 @@ const useStakingData = () => {
 		return periods;
 	}, [epochPeriod]);
 
-	const { refetch: resetVesting, data: vestingSchedules } = useContractRead({
+	const { data: vestingSchedules } = useContractRead({
 		...rewardEscrowContract,
 		functionName: 'getVestingSchedules',
 		args: [walletAddress ?? undefined, 0, 1000],
-		watch: false,
+		scopeKey: 'staking',
+		watch: true,
 		enabled: !!walletAddress,
-		select: (data) => data.filter((d) => d.escrowAmount.gt(0)),
+		select: (data) => (data as VestingEntry[]).filter((d) => d.escrowAmount > 0),
```

Comment:
Is this definitely a number? Seems like it would be a BigNumber

Reply:
Yes. This should be `BigNumber`. The change has been made.